dependency_map_view/hooks.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+Post-Installation Hook for Dependency Map View
+
+This module contains the post-installation hook that automatically adds the
+'dependency_map' view mode to all existing window actions in the system.
+
+The hook ensures that the dependency map view is available across all models
+without requiring manual configuration for each action.
+"""
+
+
+def post_init_hook(env):
+    """
+    Add dependency_map view mode to all existing window actions.
```

Comment:
I see it more as an optional action. I don't think all people want to use this new view in all models.